cmd/infra/azure/create.go

```diff
@@ -165,14 +165,14 @@ func (o *CreateInfraOptions) Run(ctx context.Context, l logr.Logger) (*CreateInf
 	} else {
 		// Create a resource group for network security group
 		nsgResourceGroupName = o.Name + "-nsg"
-		nsgRG, msg, err := createResourceGroup(ctx, o, azureCreds, nsgResourceGroupName, subscriptionID)
+		nsgResourceGroupName, msg, err = createResourceGroup(ctx, o, azureCreds, nsgResourceGroupName, subscriptionID)
```

Comment:
Is this not the same name as above?

Reply:
nope when we create a resource group name we also have a unique identifier as a suffix